yoti-sdk-api/src/test/java/com/yoti/api/client/docs/session/create/filters/DocumentRestrictionsFilterTest.java

```diff
@@ -92,4 +92,24 @@ public void shouldAcceptPreBuiltDocumentRestriction() {
         assertThat(result.getDocuments(), containsInAnyOrder(documentRestrictionMock));
     }
 
+    @Test
+    public void shouldSetAllowExpiredDocumentsFlag() {
+        DocumentRestrictionsFilter result = DocumentRestrictionsFilter.builder()
+                .forWhitelist()
+                .withAllowExpiredDocuments(false)
+                .build();
+
+        assertThat(result.getAllowExpiredDocuments(), is(false));
+    }
+
+    @Test
+    public void shouldSetAllowExpiredDocumentsFlagToTrueByDefault() {
```

Comment:
The SDK's should _not_ provide default values, this is handled by our backend services.  Please use a `Boolean` object here, so the property is omitted when serializing to JSON so the backend handles default values.

Reply:
I see...
Sorry, I was puzzled by the example
`"allow_expired_documents": true // THIS IS NEW, optional, default to true` in the epic. That's the requirement for our service, not for SDK

Fixed